### DISCUSSION TITLE
Search result showing a list column, how?

### DISCUSSION THREAD

**Discussion by iamhuman2414** [microsoft-search/pnp-modern-search#2021] [Q&A] [answered]
Newbie here! TIA for any assistance.

I have a very simple Sharepoint list -- with three columns ( Title, Report Number and Link ).
I added Search Box and Search Results web parts on my search page. I was able to get them to work. 
When I do a search, it only shows me Title information in the results.
How do I show my other columns (Report Number and Link) in the Search Results?? 
I know that this will be a Custom Template, but Im struggling on how to show them in the search results, can you please help? Thanks!

**Comment by wobba** [comment, marked as answer]
See #1974 to get you started.

**Reply by iamhuman2414** [reply]
I got it to work, thanks! That was really helpful. 
Now, I have another problem, https://github.com/microsoft-search/pnp-modern-search/discussions/2024

**Reply by iamhuman2414** [reply]
I figured out a workaround, I changed the type from URL to Single Line of Text, that did the trick. But would be nice to know how to do this for URL-type columns
